src/index.js

```diff
@@ -1,38 +1,35 @@
 import './style.css';
+import CreateList from './createList.js';
 
-const taskArray = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
-const inputList = document.getElementById('input-List');
+const input = document.getElementById('input-List');
 
-const createItem = (item = {
-  task: inputList.value,
-  index: taskArray.length,
-  completed: false,
-}) => {
-  taskArray.push(item);
-  localStorage.setItem('tasks', JSON.stringify(taskArray));
-  window.location.reload();
-};
-
-inputList.value = '';
-inputList.addEventListener('keydown', (event) => {
-  if (event.key === 'Enter') {
-    createItem(inputList.value);
+input.addEventListener('keydown', (event) => {
+  if (event.key === 'Enter' && input.value !== '') {
+    new CreateList().create(input.value);
+    input.value = '';
   }
 });
 
-function displayItems() {
-  let tasks = '';
-  for (let i = 0; i < taskArray.length; i += 1) {
-    tasks += ` <div class="item" >
-    <div class="input-Content">
-    <div>
-        <input type="checkbox" class="complete" name="completed" />
-        <span disabled>${taskArray[i]}</span>
-    </div>
-       
-        </div>`;
-  }
-  document.querySelector('#todo-Collections').innerHTML = tasks;
-}
+const reload = () => {
+  const collected = JSON.parse(localStorage.getItem('text'));
+  collected.forEach((item) => {
+    new CreateList().create(item.text);
+  });
+};
+
+const taskArray = JSON.parse(localStorage.getItem('text'));
+const clearCompleted = document.querySelector('#clear-Completed');
+clearCompleted.addEventListener('click', () => {
+  const checks = document.querySelectorAll('input[type=checkbox]');
+  const indexesToDelete = [];
+  checks.forEach((checkbox, i) => {
+    if (checkbox.checked) {
+      indexesToDelete.push(i);
+    }
+  });
+  const filteredItemsArray = taskArray.filter((i) => !indexesToDelete.includes(i));
+  localStorage.setItem('items', JSON.stringify(filteredItemsArray));
+  window.location.reload();
+});
```

Comment:
Well done on the `Clear all completed button` 💯  but it only refreshes the page and doesn't, delete the `completed` todos, or if they're being deleted, the new array might not be getting persisted to `localStorage` 🤔 , kindly make sure to address that as it is one of the requirements:
> A function for the "Clear all completed" button using the Array.filter() method is implemented.

Keep going and happy hacking 👨🏽‍💻